Squeue backend unit tests fail on Dom

```
____________________________________________________________________________ TestSqueueJob.test_poll _____________________________________________________________________________

self = <unittests.test_schedulers.TestSqueueJob testMethod=test_poll>

    @fixtures.switch_to_user_runtime
    def test_poll(self):
        self.setup_user()
        self.parallel_cmd = 'sleep 2'
        self.testjob.prepare(self.commands, self.environs)
        self.testjob.submit()
>       self.assertFalse(self.testjob.finished())
E       AssertionError: True is not false

unittests/test_schedulers.py:161: AssertionError
```

--- a/reframe/core/schedulers/slurm.py
+++ b/reframe/core/schedulers/slurm.py
@@ -19,8 +19,6 @@ def slurm_state_completed(state):
         'BOOT_FAIL',
         'CANCELLED',
         'COMPLETED',
-        'CONFIGURING',
-        'COMPLETING',
         'DEADLINE',
         'FAILED',
         'NODE_FAIL',
@@ -33,6 +31,8 @@ def slurm_state_completed(state):
 
 def slurm_state_pending(state):
     pending_states = {
+        'COMPLETING',
+        'CONFIGURING',
         'PENDING',
         'RESV_DEL_HOLD',
         'REQUEUE_FED',

